script/Arcueid/Day 11/11_00_ARC11_1B.txt

```diff
@@ -51,67 +51,70 @@ I'm like a kite whose string has been cut
 [sha:9d7dd7b70377eb3c7e3830667dd8862ab7fcc9c4]{
 -- Page 34, Offset 29575.
 -- 街を守るという正義感も、
-The sense of duty to protect the town,
+The duty I felt to protect this town―――
 }
 [sha:788cc537d5de7f80d44aefb2ded07d5e37a7d429]{
 -- Page 34, Offset 29576.
 -- 吸血鬼を倒す使命感も消えてしまった。
-and to bring down the vampires, is gone.
+To defeat the vampire―――is gone.
 }
 [sha:38a2550bc8d5f97f17231bc457d858a5aede17dc]{
 -- Page 34, Offset 29577.
 -- 所詮、俺の行動原理はその程度だった。
-After all, if this is all it took for me to abandon my principles,
+My principles amounted to nothing, in the end.
 }
 [sha:7ed42a7afdee61886b941e7156b3126a9662a490]{
 -- Page 34, Offset 29578.
 -- なら、もういい。
-then it didn't matter to me to begin with.
+In which case... I should let it all go.
 }
 [sha:803c0c8be2764a5f51450476350654c139d6f6f7]{
 -- Page 34, Offset 29579.
 -- ここで目を閉じるのが、まっとうな人間らしい選択だ。
-Closing my eyes here would be the proper, human choice.
+Pretending none of it ever happened would be the proper choice to make, as a human.
 }
 [sha:11e3dcbf4a4c7ff898366082b9442162b8c4e88b]{
 -- Page 35, Offset 29580.
 -- あいつも消えた。
-She's also gone from my head.
+That woman is gone.
 }
 [sha:0633097c39081c91ac3b6866211302ebccd77063]{
 -- Page 35, Offset 29581.
 -- ノエル先生も消えた。
-Noel-sensei too.
+Noel-sensei is gone.
 }
 [sha:0dd8ecad2d1d1c3ede26a507784fae9e5a41ee13]{
 -- Page 35, Offset 29582.
 -- だいたい―――
-In their place―――
+And besides―――
 }
 [sha:4a83739e7f2a73897137fca48ac386c3d5be8169]{
 -- Page 35, Offset 29583.
 -- 吸血鬼なんて、俺の手には余りすぎる。
-That vampire stuff was all too much for me to handle.
+I was in over my head with all that vampire business anyway.
 }
 [sha:283f7886c0ae92a1201cf6a947ca2488cb8a559b]{
 -- Page 35, Offset 29584.
 -- 『人間にとって善い吸血鬼』はいない、と代行者は言った。
-That Executor said that there is no such thing as a vampire who's good for humans.
+That Executor said that there's no such thing as a good vampire.
 }
 [sha:15ba75cde3cbc583619271a8a94d27637b5b73fe]{
 -- Page 35, Offset 29585.
 -- あの姿を見て、まだ俺は信じるというのか。
-Do I still believe that after seeing her?
+// Reading this as "can I still believe that _there are good vampires_,
+// even after seeing _Arc have a heated vampire moment_?"
+// Since a) seems internally consistent and b) they flash the arc beast CG
+Can I still believe that there are, even after seeing her like that?
```

Comment:
I think "seeing Arcueid" is necessary here because the previous subject was Noel and it makes it a little muddy